common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1295,6 +1301,14 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val PUSH_TIMEOUT_CHECK_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.push.timeoutCheck.interval")
+      .categories("common")
+      .doc("Interval for checking push data timeout.")
+      .version("0.3.0")
+      .timeConf(TimeUnit.MILLISECONDS)
+      .createWithDefaultString("3s")
```

Comment:
3s is too short compared to PUSH_DATA_TIMEOUT(120s). I think we can set it default to PUSH_TIMEOUT_CHECK_INTERVAL / 2

Reply:
DOne